numpy/lib/financial.py

```diff
@@ -271,20 +279,20 @@ def nper(rate, pmt, pv, fv=0, when='end'):
     use_zero_rate = False
     with np.errstate(divide="raise"):
         try:
-            z = pmt*(1.0+rate*when)/rate
+            z = pmt*(1+rate*when)/rate
         except FloatingPointError:
             use_zero_rate = True
 
     if use_zero_rate:
-        return (-fv + pv) / (pmt + 0.0)
+        return (-fv + pv) / (pmt + 0)
```

Comment:
+0 seems pointless now

Reply:
I concur, I'll remove all occurences in the file.